README.md

```diff
@@ -1,2 +1,16 @@
 # coding-school
-A place to use when teaching someone how to start with software development 
+
+A place to use when teaching someone how to start with software development ..
```

Comment:
Hey lets remove this unnnecesary dots at the end please

Reply:
done